roles/openshift_node/tasks/main.yml

```diff
@@ -37,7 +37,8 @@
 - name: Save firewalld port for OpenShift
   firewalld: port=10250/tcp permanent=true state=enabled
 
+  # Always bounce service to pick up new credentials
 - name: Enable OpenShift
-  service: name=openshift-node enabled=yes state=started
+  service: name=openshift-node enabled=yes state=restarted
```

Comment:
You could also factorise the declaration of the firewalld in a more concise syntax like this:

```
- name: Open firewalld port for etcd embedded in OpenShift
  firewalld: port={{ item[0] }} permanent={{ item[1] }} state=enabled
  with_nested:
    - [ 4001/tcp, 8080/tcp, 8443/tcp, 8444/tcp ]
    - [ true, false ]
```

Port 8444 is used for web console